code/modules/mob/living/carbon/human/species/golem.dm

```diff
@@ -314,9 +315,9 @@
 	name = "Wood Golem"
 	golem_colour = rgb(158, 112, 75)
 	skinned_type = /obj/item/stack/sheet/wood
-	species_traits = list(NO_BLOOD, IS_PLANT) // Refactor into biotype
 	//Can burn and take damage from heat
```

Comment:
I assume `MOB_PLANT` infers `NO_BLOOD`?

Reply:
inheritance. Golems have `NO_BLOOD` by default. The only reason this is here is because they had the `IS_PLANT` trait.